csm/core/agent/api.py

```diff
@@ -151,7 +151,8 @@ def error_response(err: Exception, request) -> dict:
             resp["message"] = f'{str(err)}'
 
         audit = CsmRestApi.http_request_to_log_string(request)
-        if hasattr(request,"session"):
+        if ((hasattr(request, "session") and request.session is not None) and
```

Comment:
@tb-s  Please take a look at https://github.com/Seagate/cortx-manager/pull/54/files
During some rebase conflict my proposal was lost in dev.

Could you please change this code accordingly? For example:
if getattr(request, "session", None) is not None:
  if getattr(request.session, "credentials", None) is not None:

Reply:
@IgorPivovarovDSR I have just updated this snippet. Please check.